application/electron/src/controllers/files.parsers/file.parser.dlt.ts

```diff
@@ -17,6 +17,16 @@ import { Subscription } from "../../tools/index";
 import Logger from "../../tools/env.logger";
 import ServiceNotifications, { ENotificationType } from "../../services/service.notifications";
 
+interface IDLTFilters {
+    [key: string]: string[];
+}
+
+interface IDLTOptions {
+    logLevel: number;
+    filters: IDLTFilters;
+    fibexFilePath?: string;
+}
+
```

Comment:
why is this defined in 2 places? should only exist once

Reply:
I would say it's a more complex issue. And it's good we can resolve it white it's "small". We have "indexer-neon", we have some common things between render and process (front-end and back-end). Many common things (like interfaces) are shared between render and process via IPC message-classes. But not all, and DLT is a good case.

Now I've found in solution (and in render, and in process) direct references to indexer. Like `../../../apps/indexer-neon`. I believe it's wrong, because it breaks namespaces of parts of solution. 

In another hand, we have to share some interfaces between render and indexer-neon (for the process it isn't a problem, because indexer-neon there is a node module). The best way I think - it's creating "outside" folder `common` and subfolder `interfaces`. Also into `common`, folder `ipc` should be moved. 

it will look like
```
application
└── common
    ├── interfaces
    └── ipc
```

In this case, we can avoid from direct references to `indexer-neon` from render.